src/pages/sessions/[id].tsx

```diff
@@ -0,0 +1,236 @@
+  description: string
+  sessionType: string | null
+  sessionLevel: string | null
+  addEventToCalendarLink: string
+  speaker: {
+    fullName: string
+    profilePicture: string
+    bio: string
+    tagLine: string
+    twitterLink: string | null
+  }
+}
+
+const allSessionPaths = [
+  { params: { id: 'A3-L' } },
+  { params: { id: 'B3-L' } },
+  { params: { id: 'A4-S' } },
+  { params: { id: 'B4-S' } },
+  { params: { id: 'A5-SP' } },
+  { params: { id: 'A6-SP' } },
+  { params: { id: 'A7-L' } },
+  { params: { id: 'B7-L' } },
+  { params: { id: 'A8-S' } },
+  { params: { id: 'B8-S' } },
+  { params: { id: 'A9-S' } },
+  { params: { id: 'B9-S' } },
+  { params: { id: 'A10-S' } },
+  { params: { id: 'B10-S' } },
+  { params: { id: 'A11-S' } },
+  { params: { id: 'B11-S' } },
+  { params: { id: 'A12-S' } },
+  { params: { id: 'B12-S' } },
+  { params: { id: 'A13-S' } },
+  { params: { id: 'B13-S' } },
+  { params: { id: 'A14-S' } },
+  { params: { id: 'B14-S' } },
+  { params: { id: 'LT1' } },
+  { params: { id: 'LT2' } },
+  { params: { id: 'LT3' } },
+  { params: { id: 'LT4' } },
+  { params: { id: 'LT5' } },
+  { params: { id: 'LT6' } },
+  { params: { id: 'LT7' } },
+  { params: { id: 'LT8' } }
+]
+
+export const getStaticPaths: GetStaticPaths = async () => {
+  return {
+    paths: allSessionPaths,
+    fallback: false
+  }
+}
+
+export const getStaticProps: GetStaticProps<Props> = async ({ params }) => {
+  const response = await fetch('https://sessionize.com/api/v2/3qcdixg4/view/All')
+  const parsedResult = await sessionizeViewAllSchema.safeParseAsync(await response.json())
+
+  if (!parsedResult.success) {
+    throw new Error(`Failed to parse: ${parsedResult.error}`)
+  }
+
+  const { sessions, rooms, categories, speakers } = parsedResult.data
+
+  const matchedSession = sessions.find(
+    ({ questionAnswers }) =>
+      questionAnswers.find(({ questionId }) => questionId === QUESTION_SESSION_NUMBER)?.answerValue === params?.id
+  )
+
+  if (!matchedSession) {
+    throw new Error(`No sessions found.`)
+  }
+
+  const { title, description, roomId, categoryItems, speakers: speakerIds, startsAt, endsAt } = matchedSession
+
+  const sessionId = params?.id
+  if (!sessionId || typeof sessionId !== 'string') {
+    throw new Error(`Invalid sessionId: ${sessionId}`)
+  }
+
+  const room = rooms.find(({ id }) => id === roomId)
+
+  if (!room) {
+    throw new Error(`Invalid roomId: ${roomId}`)
+  }
+
+  const roomName = room.name
+
+  const sessionLevels = categories.find(({ id }) => id === CATEGORY_SESSION_LEVEL)?.items
+  const sessionLevel = sessionLevels?.find(({ id }) => id === categoryItems[1])?.name ?? null
+
+  const sessionTypes = categories.find(({ id }) => id === CATEGORY_SESSION_TYPE)?.items
+  const sessionType = sessionTypes?.find(({ id }) => id === categoryItems[0])?.name ?? null
+
+  const calendarDatesParam = `${dayjs(startsAt).format('YYYYMMDD[T]hhmmss')}/${dayjs(endsAt).format(
+    'YYYYMMDD[T]hhmmss'
+  )}`
```

Comment:
24時間制でフォーマットする必要があるかもです 

```suggestion
  const calendarDatesParam = `${dayjs(startsAt).format('YYYYMMDD[T]HHmmss')}/${dayjs(endsAt).format(
    'YYYYMMDD[T]HHmmss'
  )}`
```

Reply:
ありがとうございます！

[aeae44c](https://github.com/GoCon/2023/pull/240/commits/aeae44c6ecdc86087da57191067d0b9e6631500c) にて修正しました！ (ついでにスタイルも微調整してます)